specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-08-03-preview/location.json

```diff
@@ -0,0 +1,184 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ContainerServiceClient",
+    "description": "The Container Service Client.",
+    "version": "2022-08-03-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/locations/{location}/orchestrators": {
+      "get": {
+        "tags": [
+          "ContainerServices"
+        ],
+        "operationId": "ContainerServices_ListOrchestrators",
```

Comment:
Is there a desire from the RP team to deprecate this API and/or move it?

I'm just thinking that if it hasn't been being included in the various SDKs for years now, _and_ we want to move towards deprecation, it might be better to continue to leave it out of the SDK.

OTOH, if there's no plan to deprecate (or there is but we just need it in the meantime), then doing this makes sense.